pkg/util/secret.go

```diff
@@ -19,6 +19,9 @@ func GetTargetSecret(client kubeclient.Interface, secretReferences []corev1.Obje
 		klog.V(2).Infof("checking secret: %s/%s", targetNamespace, objectReference.Name)
 		secret, err := client.CoreV1().Secrets(targetNamespace).Get(context.TODO(), objectReference.Name, metav1.GetOptions{})
 		if err != nil {
+			if apierrors.IsNotFound(err) {
+				continue
+			}
```

Comment:
outer functions or outer callers will check whether err is NotFound or not, so this is redundant?

Reply:
No, the `continue` here means we go through all the secrets.

Reply:
It is not redundant, one SA will correspond to multiple secrets, of which only one may be available, here is an attempt to fix this situation

Reply:
oh, got it, I know. thanks 😄